src/app.jsx

```diff
@@ -592,6 +768,14 @@ class Application extends React.Component {
 
     componentWillUnmount() {
         cockpit.removeEventListener("locationchanged", this.onNavigate);
+
+        // Cleanup DBus subscriptions
+        this.state.users.forEach(user => {
+            if (user.dbus) {
+                user.dbus.subscription.remove();
+                user.dbus.client.close();
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2266-0a892840-20250919-141935-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)